plugin.video.youtube/resources/lib/youtube/helper/yt_specials.py

```diff
@@ -51,17 +56,22 @@ def _process_browse_channels(provider, context, re_match):
     guide_id = context.get_param('guide_id', '')
     if guide_id:
         json_data = provider.get_client(context).get_guide_category(guide_id)
+        if not v3.handle_error(provider, context, json_data):
+            return False
         result.extend(v3.response_to_items(provider, context, json_data))
         pass
     else:
         json_data = provider.get_client(context).get_guide_categories()
+        if not v3.handle_error(provider, context, json_data):
+            return False
         result.extend(v3.response_to_items(provider, context, json_data))
         pass
 
     return result
 
 
 def _process_disliked_videos(provider, context, re_match):
+    provider.set_content_type(context, kodion.constants.content_type.EPISODES)
```

Comment:
The override view that is configured during the setup wizard if a supported skin is installed or in settings. It's been there a long time. As I understood it, different views are available for different content types, episodes fits as far as views / youtube information per video for the most part which is why I'm guessing this is what was chosen. This change would cancel out that option for all these users that are used to it. To keep the option with this change would require gathering of view id per skin for video content type which may not have the view users are now accustomed to. If you need it changed, I don't understand but I will change it.